nested_kvm.yml

```
@@ -37,7 +37,7 @@
       shell: "cat /sys/module/kvm_intel/parameters/nested"
       changed_when: false
       register: result
-      failed_when: "'Y' not in result.stdout_lines"
+      failed_when: "'Y' not in result.stdout_lines or '1' not in result.stdout_lines"
```

```suggestion
      failed_when: "'Y' not in result.stdout_lines and '1' not in result.stdout_lines"
```
Currently when Y is in result.stdout that's (False or True) == True.

---

Absolutely, cheers